version.txt

```diff
@@ -53,7 +54,7 @@ nematode_traits.json,1.0.0
 ngreatplains-flowering-dates.json,1.0.0
 npn.py,2.1.2
 nyc_tree_count.json,1.0.0
-pantheria.py,1.3.1
+pantheria.py,1.3.2
```

Comment:
Some info should not be in this PR. Looks like you have some scripts that have been added in the version.txt.
Clean up the repo and run python version.py again.

You should also squash the commits to one. 

```
git clean -fx -d
retriever reset scripts
python version.txt
```